roles/archive-upload-ws/tasks/2_install_config.yml

```diff
@@ -0,0 +1,41 @@
+---
+
+- name: ensure archive-upload conf dir exists
+  file:
+    state: directory
+    path: "{{ archive_upload_config_root }}"
+    mode: g+rwXs
+
+#- name: ensure arteria-dsmc log dir exists
+#  file: 
+#    state: directory
+#    path: "{{ arteria_dsmc_log_dir }}"
+#    owner: arteria
+#    group: arteria
+#    mode: g+rwXs
+
+- name: deploy archive-upload app config
+  template:
+    src: archive_upload_app.config.j2
+    dest: "{{ archive_upload_config_root }}/app.config"
+
+- name: deploy archive-upload logger config
+  template:
+    src: archive_upload_logger.config.j2
+    dest: "{{ archive_upload_config_root }}/logger.config"
+
+- name: modify uppsala's supervisord conf to start archive-upload-ws 
+  ini_file: 
+    dest="{{ ngi_pipeline_conf }}/supervisord_upps.conf"
+    section="program:archive-upload-ws-{{ deployment_environment }}"
+    option=command
+    value="{{ archive_upload_env_root }}/bin/archive-upload-ws --configroot={{ archive_upload_config_root }} --port={{ archive_upload_port }}"
+    backup=no
+ 
+- name: modify uppsala's supervisord conf to start archive-upload-ws
```

Comment:
Is this meant to stay or just for the transition? I recently made some bad experiences with interdependent roles that need to be executed in a certain order... :D

Reply:
Yeah. It should, I think. We do this in a bunch of places for Irma, and if I remember correctly it wasn't as easy as just pulling in the upstream role as a dependency, because then that role would override things that OTHER roles have modified. I.e. everything would have to be refactored for that.

Reply:
Alright, I suspected something like that :)